src/core/lib/iomgr/tcp_server_posix.c

```diff
@@ -70,6 +71,8 @@
 
 #define MIN_SAFE_ACCEPT_QUEUE_SIZE 100
 
+int grpc_tcp_server_posix_expand_wildcard_addr = 0;
```

Comment:
I've been unable to find a good place to add it. The ideal situation would be to have the constructor for grpc_channel_args set a default value for the flag based on probing local hardware, and then the caller could customize the flag directly if necessary. However, it's a struct with no constructor, e.g., https://github.com/grpc/grpc/blob/master/src/cpp/server/server.cc#L295

I don't want to have to set it explicitly everywhere a grpc_channel_args is stack or heap allocated.

I need a grpc_channel_args_init (constructor, essentially) to add my code to, but none exists.